site/viamguides/src/plant-watering-device-workshop/plant-watering-device-workshop.md

```diff
@@ -7,7 +7,7 @@ status: Published
 feedback link: <https://github.com/viam-labs/viamcodelabs/issues>
 tags: Getting Started, Developer, Hardware
 
-# Plant Watering Device Workshop
+# Automatic plant watering
```

Comment:
Can I convince you to rename these 2 codelabs to something more along the lines of Plant Watering? With moisture sensors maybe?
I suspect we'll have a number of Automate/Automatic titles.